new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/TestActions.java

```diff
@@ -14,30 +21,225 @@
 
     // ----------------------   operator  ---------------------------------
 
+    /**
+     * Install WebLogic Kubernetes Operator
+     * @return true if the operator is successfully installed, false otherwise.
+     */
     public static boolean installOperator() {
         return Operator.install();
     }
 
+    /**
+     * Upgrade existing Operator release
+     * @param values map with values
+     * @return true if the operator is successfully upgraded, false otherwise.
+     */
+    public static boolean upgradeOperator(HashMap<String, String> values) {
```

Comment:
We should make a fluent API for the helm chart values

Reply:
agree

Reply:
work on fluent API for operator, domain and update the PR?